src/gui_impl/main_window.cpp

```diff
@@ -44,7 +44,7 @@ MainWindow::MainWindow(QSharedPointer<ApplicationContext> application_context)
     setUnifiedTitleAndToolBarOnMac (true);
     
     QSettings settings("SigViewer");
-    resize(settings.value("MainWindow/size", QSize(900, 800)).toSize());
+    resize(settings.value("MainWindow/size", QSize(1500, 800)).toSize());
```

Comment:
Hi Clemens, in my opinion 1200 might be a sensible option. Because I thought SigViewer ideally should be a rectangle instead of a square shape, what do you think? Also, I changed the info dialog to modeless, which I thought is more convenient to look at.